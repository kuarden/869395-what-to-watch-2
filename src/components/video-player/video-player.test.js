import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const FILM = {
  ID: 1,
  NAME: `Fantastic Beasts: The Crimes of Grindelwald`,
  IMAGE: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  URL: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
};

it(`VideoPlayer renders correctly`, () => {
  const tree = renderer.create(<VideoPlayer src={FILM.URL} isPlaying={false} poster={FILM.IMAGE} />).toJSON();
  expect(tree).toMatchSnapshot();
});
