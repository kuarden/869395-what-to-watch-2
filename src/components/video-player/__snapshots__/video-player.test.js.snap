// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer renders correctly 1`] = `
<video
  height="175"
  muted={true}
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  width="280"
/>
`;
